defaults/main.yml

```diff
@@ -42,3 +42,7 @@ sonar_db:
 sonar_store: "https://sonarsource.bintray.com/Distribution/sonarqube"
 # local download path
 download_path: "/tmp/"
+
+sonar_proxy:
```

Comment:
In my opinion role should provide splitted proxy feature by protocol:
- http
- https

And user should be able to choose  exact protocols he want.